src/_community/pluralkit.md

```diff
@@ -103,8 +94,8 @@ For more information on plurality, feel free to browse the following resources..
 {% admonition %}
 **Note:** This site belongs to a specific plural system, and much of what's written there is related to their personal
 experiences of plurality. Additionally, some articles there were written quite some time ago, and may contain vulgar
-language and outdated terminology - including terms that exist outside the plural community. Nonetheless, it remains 
-a popular and useful resource for people approaching plurality and plural systems from the outside.
+language and outdated terminology - including terms that exist outside the plural community. Nonetheless, it remains a 
+popular and useful resource for people approaching plurality and plural systems from the outside.
 {% endadmonition %}
 
 * [Fictive Identities](https://www.exunoplures.org/main/fictive-identities/) by Noël Dawkins
```

Comment:
If wanted to, other websites are out there like [Pluralpedia](https://pluralpedia.org/w/Plurality) or [Termora](https://termora.org/term/1) (Though termora's is probably better used as a bot as it just has the definitions)

Reply:
I don't really like either of those - the first one seems to contain a lot of unofficial or controversial terms, whereas the latter just seems to be a smaller take on the Blossom bot concept - a bot that ended up dying due to mismanagement and the addition of niche and controversial terms.

I think the sources here are probably best kept more general, or more academic - that way we avoid having to continuously maintain the list whenever there's an issue with a source.